roles/confluent.common/defaults/main.yml

```diff
@@ -62,3 +62,12 @@ confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/
 
 ### Set to true to indicate the archive file is remote (i.e. already on the target node) or a URL. Set to false if the archive file is on the control node.
 confluent_archive_file_remote: true
+
+### Base URL for Confluent CLI packages
+confluent_cli_repository_baseurl: "https://s3-us-west-2.amazonaws.com/confluent.cloud"
+
+### A path reference to a local archive file or URL. By default this is the URL from Confluent CLI repository.
+confluent_cli_archive_file_source: "{{confluent_cli_repository_baseurl}}/confluent-cli/archives/{{confluent_cli_version}}/{{confluent_cli_binary}}_{{(confluent_cli_version == 'latest') | ternary('', 'v')}}{{confluent_cli_version}}_{{ansible_system|lower}}_{{confluent_cli_goarch[ansible_architecture]}}.tar.gz"
```

Comment:
```suggestion
### A path reference to a local archive file or URL. By default this is the URL from Confluent CLI repository.
confluent_cli_archive_file_source: "https://s3-us-west-2.amazonaws.com/confluent.cloud/confluent-cli/archives/{{confluent_cli_version}}/{{confluent_cli_binary}}_{{(confluent_cli_version == 'latest') | ternary('', 'v')}}{{confluent_cli_version}}_{{ansible_system|lower}}_{{confluent_cli_goarch[ansible_architecture]}}.tar.gz"
```

Does the base url get used elsewhere?

Reply:
No `confluent_cli_repository_baseurl` it is used only to compose `confluent_cli_archive_file_source`. I used it the same way `confluent_common_repository_baseurl` its used, I thought it may be useful to change the repository location without touching the package archive path, e.g. if a new mirror is activated